projects/Mallard/src/screens/settings-screen.tsx

```diff
@@ -119,9 +120,14 @@ const SettingsScreen = ({ navigation }: NavigationInjectedProps) => {
     const identityData = useIdentity()
     const canAccess = useAccess()
     const [, setVersionClickedTimes] = useState(0)
-    const { signOutIdentity } = useContext(AccessContext)
+    const { signOutIdentity, iapData } = useContext(AccessContext)
 
     const versionNumber = DeviceInfo.getVersion()
+    const isLoggedInWithIdentity = !!(identityData
```

Comment:
what's the motivation for double banging this?

Reply:
Its a hangover from when I had it typed and put into an object early - Can remove